dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheTimeoutServerB/server.xml

```diff
@@ -0,0 +1,81 @@
+<server>
+
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+        <feature>jdbc-4.1</feature>
+        <feature>jndi-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+    
+    <httpEndpoint id="defaultHttpEndpoint"
+                  httpPort="${bvt.prop.HTTP_secondary}"
+                  httpsPort="${bvt.prop.HTTP_secondary.secure}"/>
+    
+     <!-- Use this config for sessionCache -->
+     <featureManager>
+        <feature>sessionCache-1.0</feature>
+    </featureManager>   
+    <httpSession maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="5s" reaperPollInterval="30"/>
+    <httpSessionCache libraryRef="HazelcastLib" writeContents="ALL_SESSION_ATTRIBUTES">
+        <properties hazelcast.config.location="file:${shared.resource.dir}/hazelcast/hazelcast-localhost-only.xml"/>
+    </httpSessionCache>
+    
+    <jndiEntry jndiName="jcache/hazelcast.config" value="${shared.resource.dir}/hazelcast/hazelcast-localhost-only.xml"/>
+
+    <!-- Use this config for testing sessionDatabase 
+    <featureManager>
+        <feature>sessionDatabase-1.0</feature>
+    </featureManager>
+   
+    <httpSessionDatabase id="FATDB" dataSourceRef="DerbyDS"/>
+    
+    <httpSession storageRef="FATDB" maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="5s" reaperPollInterval="30"/>
+-->
+    <application location="sessionCacheApp.war">
+        <classloader commonLibraryRef="HazelcastLib"/>
+    </application>
+    
+    <library id="HazelcastLib">
+        <file name="${shared.resource.dir}/hazelcast/hazelcast.jar"/>
+    </library>
+
+	<library id="DerbyLib">
+	    <fileset dir="/Users/mjswatos/Programming/Derby/db-derby-10.11.1.1-bin/lib"/>
+	</library>
+
+    <!-- Used for testing that data source can be stored in a session -->
+	<authData id="DerbyAuth" user="userName" password="userPassword"/>    
```

Comment:
Remove or comment out the dataSource config (maybe move it to the other commented out code block for sessionDatabase)